setup.cfg

```diff
@@ -34,7 +34,6 @@ python_requires = >= 3.8
 install_requires =
     matplotlib
     networkx
-    rdkit>=2021.*
```

Comment:
Can we just do `rdkit>=2021` or suitable minimum version here? (we need to do a cleanup, but it'd be great for folks not tot show up with old versions of rdkit and get confused why everything is falling apart).